xcode/Subconscious/Shared/Config.swift

```diff
@@ -17,6 +17,8 @@ struct Config: Equatable, Codable {
     var appTabs = false
     var addressBook = false
     var addByQRCode = true
+    var userProfile = true
```

Comment:
Will set to false before merging, might be worth checking this brach out and test driving it with the flag on for review.